console/src/view/tasks.rs

```diff
@@ -108,7 +108,7 @@ impl List {
                 let task = task.borrow();
 
                 let mut row = Row::new(vec![
-                    Cell::from(id_width.update_str(task.id_hex()).to_string()),
+                    Cell::from(id_width.update_str(task.id()).to_string()),
```

Comment:
err, whoops, i messed this up while merging:
```suggestion
                    Cell::from(id_width.update_str(task.id().to_string())),
```